src/caret_analyze/architecture/architecture.py

```diff
@@ -372,6 +387,19 @@ def update_message_context(self, node_name: str, context_type: str,
 
     def insert_publisher_callback(self, node_name: str,
                                   publish_topic_name: str, callback_name: str) -> None:
+        """
+        Insert the association of callback with publisher in "node_name" node.
+
+        Parameters
+        ----------
+        node_name : str
+            name of target node
+        publish_topic_name : str
+            name of publish topic to insert publisher callback
```

Comment:
```suggestion
            topic name of target publisher into which callback is inserted
```

Reply:
Fix it in [6cf0285](https://github.com/tier4/CARET_analyze/pull/300/commits/6cf0285fa738613d5ed20fe836eed60ce682205c).